flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/source/utils/OracleConnectionUtils.java

```diff
@@ -81,17 +83,30 @@ public static RedoLogOffset currentRedoLogOffset(JdbcConnection jdbc) {
     }
 
     public static List<TableId> listTables(
-            JdbcConnection jdbcConnection, RelationalTableFilters tableFilters)
+            JdbcConnection jdbcConnection, OracleSourceConfig oracleSourceConfig)
             throws SQLException {
         final List<TableId> capturedTableIds = new ArrayList<>();
-
+        List<String> tableList = oracleSourceConfig.getTableList();
         Set<TableId> tableIdSet = new HashSet<>();
-        String queryTablesSql =
-                "SELECT OWNER ,TABLE_NAME,TABLESPACE_NAME FROM ALL_TABLES \n"
-                        + "WHERE TABLESPACE_NAME IS NOT NULL AND TABLESPACE_NAME NOT IN ('SYSTEM','SYSAUX')";
+        StringBuilder queryTablesSql =
+                new StringBuilder(
+                        "SELECT OWNER ,TABLE_NAME,TABLESPACE_NAME FROM ALL_TABLES \n"
+                                + "WHERE TABLESPACE_NAME IS NOT NULL AND TABLESPACE_NAME NOT IN ('SYSTEM','SYSAUX')");
+        if (tableList != null && !tableList.isEmpty()) {
+            String schema = jdbcConnection.connection().getSchema();
+            StringJoiner stringJoiner = new StringJoiner(",");
+            for (String tableId : tableList) {
+                String tableName = tableId.replaceFirst(schema + "\\.", "");
+                stringJoiner.add("'" + tableName + "'");
```

Comment:
What if the user use regular expressions in the table names?
Maybe we need a test to cover this situation.